portal/config.py

```diff
@@ -37,14 +36,19 @@ class BaseConfig(object):
         'redis://localhost:6379'
     )
 
+    REDIS_DB = os.environ.get(
+        'REDIS_DB',
+        0
+    )
+
     ANONYMOUS_USER_ACCOUNT = True
     CELERY_BROKER_URL = os.environ.get(
         'CELERY_BROKER_URL',
-        REDIS_URL + '/0'
+        '{}/{}'.format(REDIS_URL, REDIS_DB)
```

Comment:
I think this would break if `REDIS_URL` was set as an env var and already included a DB eg:
```
REDIS_URL = 'redis://hostname:password@example.com:1234/9'
```